practice-app/back-end/routes/searchtrendforproduct.js

```diff
@@ -0,0 +1,59 @@
+
+var express = require('express');
+var router = express.Router();
+
+
+router.get('/', function(req, res, next) {
+
+  //create a Twitter object with tokens
+  var Twitter = require('twitter');
+  var key = require('./keys.js');
+  var Twit = new Twitter(key);
+
+  var today = new Date();
+  var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate(); //date of today
+  var yesterday = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+(today.getDate()-1); //date of yesterday
+
+  var product = req.query.product; //get product name from URL
+
+  //parameter declarations that are going to be used in search
+  var params = {
+
+    q : product,
+    count : 100, 
+    result_type: 'recent',
+    since : yesterday,
+    until : date
+  };
+
+  var tweetList = {};   //tweets are going to be stored here
```

Comment:
It is not used(you are never reading the data), (vscode should give a warning I think)

Reply:
It was used inside the loop but since loop is deleted there is no need for it